modules/op-configuring-retention-policy-results.adoc

```diff
@@ -0,0 +1,37 @@
+// This module is included in the following assembly:
+//
+// * cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-retention-policy-results_{context}"]
+= Configuring retention policy for {tekton-results}
+
+By default, {tekton-results} stores pipeline runs, task runs, events, and logs indefinitely. This leads to an unnecesary use of storage resources and can affect your database performance.
```

Comment:
It seems like it's used like this widely already, but normally we try to lower case when we're talking in general about API objects. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

So if you're specifically talking about the object, it'd be `TektonResult` in backticks. Or then just "tekton result" in general.

Again, don't need to change for this PR since it's already like that throughout the assembly. Just something to consider in the future if all of these things really need to be capitalized, or if they could follow core OCP's standard way of lowercasing general references